### ISSUE TITLE
Custom location of config files

### ISSUE THREAD

**Issue by jcgutier** [dalance/procs#394]
Can I set an environmental variable with a custom config location ? or cli parameter?

**Comment by steffenbusch** [comment]
+1 for the cli parameter. That would allow to have multiple procs aliases using different configuration files such as:
```bash
alias procs_large='procs --config ~/.config/procs/large.toml'
alias procs_small='procs --config ~/.config/procs/config.toml'
```

**Comment by dalance** [comment]
I added `--config` option.

**Comment by dalance** [comment]
I released v0.14.0.
